client/components/domains/register-domain-step/index.jsx

```diff
@@ -65,10 +66,16 @@ let lastSearchTimestamp = null;
 let searchCount = 0;
 let recordSearchFormSubmitWithDispatch;
 
+const testGroup = abtest( 'domainSuggestionTestV5' );
```

Comment:
Just a sanity check - won't this cause the user to get assigned a variation too early?

Reply:
Yes, @klimeryk. Thanks! @matthusby I totally missed this. This (and the following `if`) should be done inside the class.